### DISCUSSION TITLE
Why is meta:register never called?

### DISCUSSION THREAD

**Discussion by uyamazak** [nuxt/nuxt#16964] [Questions] [answered]
Hi, the lifecycle hook `meta:register` is described in the following document and codes, but seems to me it is not called for.

https://v3.nuxtjs.org/api/advanced/hooks#lifecycle-hooks

https://github.com/nuxt/framework/search?q=meta%3Aregister

reproduction
https://stackblitz.com/edit/github-pqrv3d?file=plugins/hook.ts

in console
```
app:created
app:rendered
app:created
app:rendered
app:created
app:rendered
app:created
app:rendered
✔ Vite server built in 195ms 
```

I found this hook because I wanted to create a module or plugin to add external scripts to the meta of all pages.

Is this hook still under development? Or will it be moved to useHead, etc. and no longer implemented?

**Comment by danielroe** [comment, marked as answer]
Thanks for spotting it! I've added a brief explanation in https://github.com/nuxt/framework/pull/7130. If you want to add external scripts, you should be able to do so with `useHead` within a Nuxt plugin. (It is also possible to hook into the render result, but it's difficult to know how best to advise you without more info.)

**Reply by uyamazak** [reply]
@danielroe
Thanks you for your response.
I'm already doing those with useHead(), so now I have a clearer idea of how to do it!
